File: EvenNumbers.cs
using System.Collections.Generic;

namespace Arithmetic_CS
{
    class EvenNumbers
    {
        private int start; // Our starting point
        private int stop; // where we will stop
        private List<int> list_of_primes; // We will house our gathered prime numbers here.

        // Our constructor
        public EvenNumbers(int first, int last)
        {
            start = first;
            stop = last;
            list_of_primes = new List<int>();
        }

        public List<int> prepResult()
        {
            /*
            * Loop from start to stop and rip out even numbers;
             */
            while (start <= stop)
            {
                if (start % 2 == 0)
                { // modulo(%) is explained later
                    list_of_primes.Add(start);
                }
                start = start + 1; // increase 'start' by 1
            }
            return list_of_primes;
        }
    }
}

File: EvenNumbers.java
package arithmetic;

public class EvenNumbers {
    private int start; // Our starting point
    private final int stop; // where we will stop
    private String result; // Store result here.
    
    // Our constructor
    public EvenNumbers(int first, int last) {
        start = first;
        stop = last;
        result = "Even numbers between " + first + " and " + last + " are: \n";
    }

    public String prepResult() {
        /*
        * Loop from start to stop and rip out even numbers;
         */
        while (start <= stop) {
            if (start % 2 == 0) { // modulo(%) is explained below
                result = result + start + "; ";
            }
            start = start + 1; // increase start by 1
        }
        return result;
    }
}
